src/ukify/ukify.py

```diff
@@ -466,6 +466,18 @@ def sign(input_f: str, output_f: str, opts: UkifyConfig) -> None:
     def verify(opts: UkifyConfig) -> bool:
         raise NotImplementedError()
 
+    @staticmethod
+    def get(name) -> type['SignTool']:
```

Comment:
The name is a bit generic for an alternative constructor, how about
```suggestion
    def from_string(name) -> type['SignTool']:
```